cmd/ghwc/commands/cpu.go

```diff
@@ -43,12 +43,12 @@ func showCPU(cmd *cobra.Command, args []string) error {
 				// pretty-print the (large) block of capability strings into rows
 				// of 6 capability strings
 				rows := int(math.Ceil(float64(len(proc.Capabilities)) / float64(6)))
-				for row := 1; row < rows; row = row + 1 {
-					rowStart := (row * 6) - 1
+				for row := 0; row < rows; row = row + 1 {
```

Comment:
this change seems to have some merit, but it should be moved on its own commit (or possibly on its own PR) since it seems more refactoring/cleanup than actually part of the core concept of this PR